src/main/java/controller/planning/EventPlanningController.java

```diff
@@ -13,13 +14,16 @@
 
 import java.sql.SQLException;
 import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
 
 public class EventPlanningController {
     private EventPlanning eventPlanningWindow;
     private DBInitializer tablesInitializer;
     private UserRepository userRepository;
     private PartyRepository partyRepository;
+    private List<Long> users_id;
+    private Long party_id;
```

Comment:
Why is it field?

Reply:
Oh i see. We discussed this and with fields like this i need to know every detail of the code to know what's the status of the fields. `createMethods` should return `Ids` instead to ensure we have readable data flow here

Reply:
Make use of local variables which are deleted for you at the end of the scope. Fields aren't deleted implicitly anywhere